generators/app/templates/options/storybook/copy/example/button.stories.js

```diff
@@ -0,0 +1,22 @@
+import buttonTemplate from './button.twig';
+import buttonData from './button.json';
+import '@atoms/button/button.css';
+
+export default {
```

Comment:
I see this is duplicate of https://github.com/axelerant/kashmir/pull/240/files#diff-c50492faa8acc77079cf53d996ac7e1474ccb31ad3f19ae22724ce2e36b35d3aR1

Can we make this change in the #240 PR itself?

Reply:
@swarad07 Sure, I Will check if I can update that PR now with this change. It's not a duplicate though as this file path has changed and updated accordingly in `index.js`

Reply:
@swarad07 @skippednote This is now updated as part of https://github.com/axelerant/create-drupal-theme/pull/240/commits/dea3dc1a9bbb6971e1e06952ed4b0ffbc25a330c I am closing this MR as the Pattenlab CSS linkage needs more attention and I will supply another specific PR for it once I get it working.